CHANGELOG

```diff
@@ -12,7 +12,7 @@
  * removed Twig_Autoloader (use Composer instead)
  * removed `true` as an equivalent to `html` for the auto-escaping strategy
  * removed pre-1.8 autoescape tag syntax
- * dropped support for PHP 5.2, 5.3, 5.4, and 5.5
+ * dropped support for PHP 5.x
```

Comment:
Actually, as stof mentioned, you already implement this [in the Symfony travis.yml](https://github.com/symfony/symfony/blob/master/.travis.yml#L23).